site/layouts/partials/theme-toggler.html

```diff
@@ -0,0 +1,34 @@
+{{- $isExamples := eq .Layout "examples" -}}
+<button class="btn {{ if $isExamples }}btn-bd-primary{{ else }}btn-link nav-link px-0 px-lg-2{{ end }} py-2 px-0 px-lg-2 dropdown-toggle d-flex align-items-center"
```

Comment:
```suggestion
<button class="btn {{ if $isExamples }}btn-bd-primary{{ else }}btn-link nav-link px-0 px-lg-2{{ end }} py-2 dropdown-toggle d-flex align-items-center"
```

Reply:
Ah, yeah missed this when copy-pasting, thanks!